crates/ruma-common/src/events/call.rs

```diff
@@ -6,9 +6,77 @@ pub mod answer;
 pub mod candidates;
 pub mod hangup;
 pub mod invite;
+#[cfg(feature = "unstable-msc2746")]
+pub mod negotiate;
+#[cfg(feature = "unstable-msc2746")]
+pub mod reject;
+#[cfg(feature = "unstable-msc2746")]
+pub mod select_answer;
+#[cfg(feature = "unstable-msc2746")]
+mod version_serde;
 
+#[cfg(feature = "unstable-msc2746")]
+use js_int::UInt;
 use serde::{Deserialize, Serialize};
 
+use crate::{serde::StringEnum, PrivOwnedStr};
+
+/// A VoIP session description.
+///
+/// With the `unstable-msc2746` feature, this is the same type as WebRTC's
+/// [`RTCSessionDescriptionInit`].
```

Comment:
And what is it without that feature? 😅

Reply:
Well, technically, without the feature, `SessionDescriptionType` is not equivalent to `RTCSdpType`, so this cannot be equivalent to `RTCSessionDescriptionInit`, even if nothing changes in this type.

But I can remove that bit if you prefer.

Reply:
But our `SessionDescriptionType` can represent an arbitrary string either way. I think it might make sense to link to the rtc stuff, but without the unstable feature reference, which I find a bit confusing.